tools/test/scripts/doxy-spellchecker/spell.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
You can also do `#!/bin/bash -eu`

Because `pipefail` is an option, it would still need to be set with `set -o pipefail`